### PR TITLE
Cinder and  Manila: Some fixes to Hitachi backend UI options

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

**Comment by sayalilunkad** [comment]
@Itxaka do you think there are some more options I could get rid of which may not need to exposed?

**Comment by Itxaka** [comment]
@sayalilunkad No idea. During both meetings with hitachi, they were not really clear to me on what should be exposed and what not. Maybe @vuntz has more up to date info regarding that?

**Comment by sayalilunkad** [comment]
@Itxaka yes @vuntz said the same so we decided to just remove the ones that are unlikely to be used so often.

**Comment by sayalilunkad** [comment]
@Itxaka @toabctl can you please check again I have added some more commits.

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml [hunk 1]
```diff
@@ -184,28 +184,18 @@
...
-              = string_field %w(volumes {{@index}} hitachi hitachi_horcm_numbers)
               = password_field %w(volumes {{@index}} hitachi hitachi_horcm_password)
-              = integer_field %w(volumes {{@index}} hitachi hitachi_horcm_resource_lock_timeout)
               = string_field %w(volumes {{@index}} hitachi hitachi_horcm_user)
               = string_field %w(volumes {{@index}} hitachi hitachi_ldev_range)
               = string_field %w(volumes {{@index}} hitachi hitachi_pool_id)
               = string_field %w(volumes {{@index}} hitachi hitachi_serial_number)
-              = string_field %w(volumes {{@index}} hitachi hitachi_target_ports)
               = string_field %w(volumes {{@index}} hitachi hitachi_thin_pool_id)
-              = string_field %w(volumes {{@index}} hitachi hitachi_unit_name)
```
Comment (vuntz) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: Are you sure about that one?

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml [hunk 2]
```diff
@@ -184,28 +184,18 @@
...
-              = boolean_field %w(volumes {{@index}} hitachi hitachi_zoning_request)
+              = string_field %w(volumes {{@index}} hitachi hitachi_target_ports)
 
               %div{ :id => "hitachi_iscsi_container_{{@index}}" }
                 = string_field %w(volumes {{@index}} hitachi hitachi_auth_method)
-                = password_field %w(volumes {{@index}} hitachi hitachi_auth_password)
-                = string_field %w(volumes {{@index}} hitachi hitachi_auth_user)
```
Comment (vuntz) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: It feels to me like `auth_method` should either also be hidden, or these two should stay visible.

(btw, if we keep them visible, usually the password is after the user ;-))

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml [hunk 3]
```diff
@@ -184,28 +184,18 @@
...
-              = integer_field %w(volumes {{@index}} hitachi hitachi_async_copy_check_interval)
-              = integer_field %w(volumes {{@index}} hitachi hitachi_copy_check_interval)
-              = integer_field %w(volumes {{@index}} hitachi hitachi_copy_speed)
-              = string_field %w(volumes {{@index}} hitachi hitachi_default_copy_method)
               = string_field %w(volumes {{@index}} hitachi hitachi_group_range)
               = boolean_field %w(volumes {{@index}} hitachi hitachi_horcm_add_conf)
-              = string_field %w(volumes {{@index}} hitachi hitachi_horcm_numbers)
               = password_field %w(volumes {{@index}} hitachi hitachi_horcm_password)
-              = integer_field %w(volumes {{@index}} hitachi hitachi_horcm_resource_lock_timeout)
               = string_field %w(volumes {{@index}} hitachi hitachi_horcm_user)
```
Comment (vuntz) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: Can you move the user above the password?

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml [hunk 4]
```diff
@@ -184,28 +184,18 @@
                "data-showit" => "iscsi", "data-showit-target" => "#hitachi_iscsi_container_{{@index}}",
                "data-showit-direct" => "true"
               = boolean_field %w(volumes {{@index}} hitachi hitachi_add_chap_user)
```
Comment (vuntz) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: Shouldn't this one go with the auth method?

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml (hunk 1 above)
Comment (sayalilunkad) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: Well they haven't specified anything in the spreadsheet about this and it is None by default so I thought it is ok to not have it.

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml (hunk 4 above)
Comment (sayalilunkad) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: yes this one can also be removed I guess..

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml (hunk 2 above)
Comment (sayalilunkad) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: ok, I think I will remove the auth_method

**Comment by rsalevsky** [comment]
LGTM, I would like to have this in M7. Can we merge?

**Pull request merged**